battle.go

```diff
@@ -165,10 +146,10 @@ type battleContext struct {
 }
 
 func (c *battleContext) getTeam() []int {
-	selfAgent := c.ActivePokemon[c.self].AgentIdx
+	selfParty := c.ActivePokemon[c.self].PartyIdx
 	var team []int
 	for _, t := range c.Context.teams {
```

Comment:
This might be cleaner if we move team to a field on Party.

Reply:
You're probably right, but lets leave it out of this PR.